src/tools/ceph-dencoder/rbd_types.h

```diff
@@ -3,13 +3,26 @@
 TYPE(librbd::journal::EventEntry)
 TYPE(librbd::journal::ClientData)
 TYPE(librbd::journal::TagData)
+using namespace librbd::journal;
+TYPE(EventEntry)
+TYPE(ClientData)
+TYPE(TagData)
```

Comment:
We already have
```
TYPE(librbd::journal::EventEntry)
TYPE(librbd::journal::ClientData)
TYPE(librbd::journal::TagData)
```
above.  Is adding
```
using namespace librbd::journal;
TYPE(EventEntry)
TYPE(ClientData)
TYPE(TagData)
```
on top of that really necessary?

Reply:
see the other comment about namespaces.